pagopa/pn-frontend
# fix(PN-10222): fix error message management

## Short description
fix error message management in pf-login and add icon and cta in errorpage

## List of changes proposed in this pull request
- Add icon in Dialog in LoginError
- Add cta button to go back to login
- Fix message making it related to the error received

## How to test
Go on with the login in step and cancel the flow in the spid credentials. Check the error message is consistent with the error code (you can see it in the url)
